app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: "Cafe") -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
The error message 'All friends should be vaccinated' is not descriptive enough. Consider including more context, such as which friend is not vaccinated, to make it more informative.